### PR TITLE
php: add new example

### PR DESCRIPTION
**Pull request by slash3b**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/php.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Start a PHP built-in web server in the current directory:
 
 `php -S {{host:port}}`
+
+- Get list of installed PHP extensions:
```
Comment (sbrl) [inline pages/common/php.md]: Perhaps this will read better?

```suggestion
- Get a list of installed PHP extensions:
```

Comment (slash3b) [reply pages/common/php.md]: yes it is, thank you

**Comment by slash3b** [comment]
thank you!

**Comment by sbrl** [comment]
np

**Pull request merged**